pyproject.toml

```diff
@@ -62,7 +62,7 @@ vulture = "==2.6"
 pylint = "==2.15.5"
 liccheck = "==0.7.2"
 flake8-copyright = "==0.2.3"
-grpcio-tools = "==1.53.0"
+grpcio-tools = "*"
```

Comment:
The ase as [my comment](https://github.com/fetchai/cosmpy/pull/380/files#r1447357058) above, with addition, that this might have wider impact - something tells me (I might be wrong here) that protobuf compiler comes with `grpcio-tools` package, what can break re-generation of python types from protobuf schema we have currently in place.

Though, I would not worry that much, we will need to rework that "old" generation mechanism to new one - the [Buf](https://buf.build/docs/introduction)